components/FAQ.tsx

```diff
@@ -1,50 +1,99 @@
 import React from "react";
 import styles from "../styles/FAQ.module.css";
-import FaqOneQuestion from "./elements/FaqOneQuestion";
+import Image from "next/image";
+
+function CloseIcon(props: any) {
+	return (
+		<div>
+			<span className={styles.closeIcon}>
+				<Image
+					src="/icons/faqClose.png"
+					alt="Free Medical Advice"
+					width={25}
+					height={25}
+					layout="fixed"
+					className={styles.iconClass}
+				/>
+			</span>
+		</div>
+	);
+}
 
 export default function FAQ() {
 	return (
 		<div className={styles.globalFaq} id="faq">
 			<h2>Frequently Asked Questions</h2>
 			<div className={styles.faqArray}>
-				<FaqOneQuestion
-					question={
-						"How long does it take to hear back from a medical doctor?"
-					}
-					answer={
-						"You should hear back from a healthcare provider on the Medex platform in a few minutes after request. If your request was sent at night, then the response time may increase to serval hours of waiting"
-					}
-				/>
-				<FaqOneQuestion
-					question={"Who is Medex for?"}
-					answer={
-						"Medex is a digital health service that provides telemedicine and telepharmacy for adults to treat their health problems quickly and effectively at any time and any place."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Who is the health care professionals in Medex?"}
-					answer={
-						"All of the healthcare team on the Medex platform are licensed healthcare professionals. Each doctor undergoeas an extensive background check and license verification process. When you message or talk with a physican on the Medex platform you can see their resume and medical license."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Can I use any pharmacy?"}
-					answer={
-						"Yes, you can just choose the address of your local pharmacy and medications will be shipped there for free."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Can I purchase medications with home delivery?"}
-					answer={
-						"OTC medications: Yes, Rx medications: No, The current position of the Chief Pharmaceutical Inspector is identical to the act and is clear - internet - order sale of prescription drugs is impossible. Pharmaceutical law regulations in force in Poland do not allow this."
-					}
-				/>
-				<FaqOneQuestion
-					question={"Is my personal data safe?"}
-					answer={
-						"Yes. We take your privacy and the security of your information very seriously. Please read our Privacy Policy for more details. "
-					}
-				/>
+				<div className={styles.faqBox}>
+					<CloseIcon />
```

Comment:
jak już dodajesz komponent CloseIcon, to znajdź jakiś sposób, żeby 

polecam `const... = useState([])`
gdzie za każdym razem CloseIcon onClick będziesz ustawiał nowy state wyżej tak, żeby togglował stan wyświetlania szczegółów danego bloku

ale uwaga to w ramach rozgrzewki array + useState, weź pod uwagę, że tego w nowym wyglądzie PL może nie być

Reply:
Ale jaki sens ma zamykanie pytanie z działu FAQ? Mogę to zaimplementować ale dla mnie jest to kompletnie nielogiczna funkcjonalność

(chodzi o ten krzyżyk)

![image](https://user-images.githubusercontent.com/45758580/134164586-d12a7f15-e415-412c-b5c3-4b52b5f42475.png)

Reply:
bo one beda chyba najpierw otwierane, chodzi o to, zeby liosta pytan byla na poczatku tylko